packages/site/site-react/src/iframebridge/IFrameBridge.tsx

```
@@ -74,14 +75,24 @@ export const IFrameBridgeContext = createContext<IFrameBridgeContext>({
     },
 });
 
+function getContentScopeFromJwt(jwt: string): unknown {
+    try {
+        const parsedJwt = decodeJwt(jwt);
+        return parsedJwt.scope;
+    } catch (e) {
+        console.error("Failed to parse content scope JWT", e);
+        return undefined;
```

Should this fail silently?